pyproject.toml

```diff
@@ -0,0 +1,84 @@
+[project]
+name = "neo"
+version = "0.12.0.dev0"
+
+authors = [
+  { name="Samuel Garcia", email="samuel.garcia@cnrs.fr" },
+  { name="Andrew Davison", email="andrew.davison@cnrs.fr" },
+  { name="Julia Sprenger", email="julia.sprenger@univ-amu.fr" },
+  { name="Michael Denker", email="m.denker@fz-juelich.de" },
+]
+
+description = "Neo is a package for representing electrophysiology data in Python, together with support for reading a wide range of neurophysiology file formats"
+
+readme = "README.rst"
```

Comment:
I presume this replaces the "long_description" field?

Reply:
description is the short one from what I understand.
For the long one. I don't known how to include yet.
I forgot also about that.
Maybe the readme replace it no ?